rosa_getting_started/osd-aws-privatelink-creating-cluster.adoc

```diff
@@ -16,4 +15,4 @@ xref:../rosa_getting_started/rosa-config-identity-providers.adoc#rosa-config-ide
 == Additional resources
```

Comment:
This heading isn't displaying

Reply:
It's in a different module: https://deploy-preview-34123--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started/osd-aws-privatelink-creating-cluster.html#additional-resources